mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp

```diff
@@ -2697,6 +2714,7 @@ struct Conv1DGenerator
         return;
       break;
     }
+    hasTensorSemantics = linalgOp.hasPureTensorSemantics();
```

Comment:
?

Reply:
You have `op` in that function, is that the linalg op? If so you could do `cast<LinalgOp>(op).hasPureTensorSemantics()` within `maybeMaskXferOp`, rather than add a new member variable.

Reply:
Thanks for the suggestion, Ben! 🙏🏻